wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,73 @@
+# WNT API Examples
+
+This folder contains the Wirepas Network Tool API Examples. These examples show how to interact with
+the server using authentication -, metadata - and real time situation connections.
```

Comment:
There are a few stray "-" in the text. autentication **-**, metadata **-**

Reply:
Fixed